validator/modules/llm_judge/__init__.py

```
@@ -0,0 +1,781 @@
+        Select evaluation model based on eval_args configuration
+        """
+        eval_model_list = eval_args.get("eval_model_list", [])
+
+        if eval_model_list:
+            # Check if all models in eval_model_list are available
+            available_eval_models = [
+                model for model in eval_model_list if model in self.available_models
+            ]
+
+            if len(available_eval_models) == len(eval_model_list):
+                selected_model = random.choice(eval_model_list)
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)